### DISCUSSION TITLE
Battery charging with a yo yo effect AC 5.0

### DISCUSSION THREAD

**Discussion by ibby24** [nathanmarlor/foxess_modbus#691] [General]
Hi
I have recently got the integration installed on my HA server. 
A few days ago I did the modbus and then followed by all the bits needed to run the agile integration. 

I have setup triggers, based on a cost price of 0.18p and then an automation that switches the inverter into force charge to charge the battery. 

Symptoms are:
The trigger runs and the automation starts however the battery charge runs to 2.4kw and then to 0 and then up and down and up and down. 

In the trigger, only one event runs and nothing else happens, I can't see anything in the logs. 

I am not sure why this is happening. Has ny one got ideas ?

**Comment by FozzieUK** [comment]
you've got too many variables there - you need to start by checking the basics by simply changing the work mode to Force Charge, monitor that and if that is ok then it's something to do with automation.

The other thing to say is you cannot use Fox schedules at the same time as using the integration, any cloud schedules override what you are setting and you will see exactly what you say as the integration sets it to do one thing, then the schedule overrides it, and back and forth - make sure you have any schedules disabled.
